apps/qsim_amplitudes.cc

```diff
@@ -34,7 +34,7 @@
 constexpr char usage[] = "usage:\n  ./qsim_amplitudes -c circuit_file "
                          "-d times_to_save_results -i input_files "
                          "-o output_files -s seed -t num_threads "
-                         "-f max_fused_size -v verbosity\n";
+                         "-f max_fused_size -v verbosity -z\n";
```

Comment:
[Optional] The trailing `-z` here without additional detail feels unhelpful, but it matches the format (as `-z` takes no argument). Do you think there would be value in adding clarification here?

(I'm kind of on the fence here, since the apps are mostly unused at this point anyways and it's documented in other places...just looking for your sentiment on this)

Reply:
I think this is OK. It's like setting a boolean flag when there is no parameter.